reference/api/tasks.md

```diff
@@ -185,6 +185,20 @@ The `details` object is set to `null` for `snapshotCreation` tasks.
 **Type**: String
 **Description**: The date and time when the task finished `processing`, whether `failed`, `succeeded`, or `canceled`, in [RFC 3339](https://www.ietf.org/rfc/rfc3339.txt) format
 
+### Summarized task object
```

Comment:
Would it be better to add this to `reference/api/overview.md#response-body`?

Reply:
Personally, I think it makes sense to have this in two places max: the asynchronous operations guide, and the tasks API reference.

(I feel less concerned about duplicating tables than duplicating prose, since if any of the keywords are changed in a future release, it is easy to locate all occurrences of the table with a `grep`. Tables are also updated less frequently than prose, and have clear start and end points.)